pronto_mia/lib/core/services/pdf_service.dart

```diff
@@ -7,16 +7,23 @@ import 'package:pronto_mia/core/services/jwt_token_service.dart';
 import 'package:url_launcher/url_launcher.dart';
 
 class PdfService {
-  // TODO: Review cache options
-  final _cacheManager = DefaultCacheManager();
+  static const String cacheKey = 'cacheManagerCacheKey';
+
+  final _cacheManager = CacheManager(
+    Config(
+      cacheKey,
+      stalePeriod: const Duration(days: 7),
+      maxNrOfCacheObjects: 20,

```

Comment:
Why 20?

Reply:
There is no specific reason why I chose 20, this number was taken from the documentation of the `cache_manager` library.
We can change it if you want.